app/javascript/packs/document-capture.jsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import { render } from 'react-dom';
 import DocumentCapture from '../app/document-capture/components/document-capture';
+import AssetContext from '../app/document-capture/context/assets';
+import I18nContext from '../app/document-capture/context/i18n';
+
+const { I18n: i18n, AssetStrings } = window.LoginGov;
```

Comment:
This relates to a review comment at https://github.com/18F/identity-idp/pull/3909#pullrequestreview-447431945 which I'll need to respond to, but the idea with renaming `LoginGov.I18n` to `i18n` was to adhere to camel-case conventions with lowercase first letter. The same could apply to `AssetStrings`.

But it depends on how we'd want to approach the naming convention, since there is some precedent in the broader ecosystem for naming fixed enumerated sets of values in this way as `PascalCase` (for example, see [Google style guide](https://google.github.io/styleguide/jsguide.html#features-objects-enums) or [TypeScript enum documentation](https://www.typescriptlang.org/docs/handbook/enums.html)).